examples/python/circuit_draw.py

```diff
@@ -29,5 +29,8 @@ def build_bell_circuit():
 # Create the circuit
 bell_circuit = build_bell_circuit()
 
-# Use the internal .draw() to print the circuit
+# Print the circuit directly to stdout
 print(bell_circuit)
+
+# Or use the internal .draw() to print the circuit
+bell_circuit.draw()
```

Comment:
Alternately, just avoid talking about internals in the comment:
 
```suggestion
# Print the circuit
print(bell_circuit)
```

Reply:
sweet. done